build.sbt

```diff
@@ -38,4 +38,22 @@ excludeDependencies ++= Seq(
 
 assemblyOutputPath in assembly := file("/tmp/conseil.jar")
 
-scalacOptions ++= ScalacOptions.common
\ No newline at end of file
+scalacOptions ++= ScalacOptions.common
+
+import complete.DefaultParsers._
+
+fork in runConseil := true
+lazy val runConseil = taskKey[Unit]("A conseil run task.")
+javaOptions in runConseil ++= Seq("-Xms512M", "-Xmx4096M", "-Xss1M", "-XX:+CMSClassUnloadingEnabled")
+fullRunTask(runConseil, Runtime, "tech.cryptonomic.conseil.Conseil")
+
+fork in runLorre := true
+lazy val runLorre = inputKey[Unit]("A lorre run task.")
+javaOptions ++= Seq("-Xmx512M", "-Xss1M", "-XX:+CMSClassUnloadingEnabled")
+runLorre := Def.inputTaskDyn {
```

Comment:
Don't know in detail how this works, but from the [docs](https://www.scala-sbt.org/1.0/docs/Faq.html#How+can+I+create+a+custom+run+task%2C+in+addition+to+%3F) I would've thought that we only needed to use `fullRunInputTask` to get the cli argument.

Why the dynamic input part?

Reply:
yeah, that would seem natural, but its sbt so just using `fullRunInputTask` does not want to read the config file 😄

Reply:
dynamic part basically allows to evaluate args and put them into the task. In regular input task you could only hardcode things.